### ISSUE TITLE
Can we have a time series forecasting & classification example using Keras-tcn

### ISSUE THREAD

**Issue by rt3722** [philipperemy/keras-tcn#50]

**Comment by philipperemy** [comment]
@rt3722 how do you want it? Do you have an example? A link?

**Comment by rt3722** [comment]
For now if you can provide a simple example using univariate time series like based on past 12 month data predict next one month , 12 historic data points and 1 period forecast. 
Thank you.

**Comment by rt3722** [comment]
I don't know if we can use keras-tcn for categorical sequence prediction like based on past 12 categorical sequences like 0 1 1 2 3 3 3 3 3 3 3 3 ==> 1 

if we can, then an example for that would also be really helpful

**Comment by philipperemy** [comment]
@rt3722 what do you mean exactly by categorical sequence prediction? That the target is a categorical variable?

**Comment by philipperemy** [comment]
> https://github.com/philipperemy/keras-tcn/pull/51/files

**Comment by philipperemy** [comment]
How about this example here?

**Comment by philipperemy** [comment]
![image](https://user-images.githubusercontent.com/4516927/56402344-e69ad000-6297-11e9-824e-69bb50fb37d9.png)

**Comment by rt3722** [comment]
it should good.

Thank you.

**Comment by rt3722** [comment]
> @rt3722 what do you mean exactly by categorical sequence prediction? That the target is a categorical variable?

I mean they are event sequences hence categorical data.
In the example that i gave we have past 12 month event sequences and based on those sequence of events we need to predict what would be the next month event

**Comment by rt3722** [comment]
this is exactly what i am looking for.

Thank you.

**Comment by rt3722** [comment]
> @rt3722 what do you mean exactly by categorical sequence prediction? That the target is a categorical variable?

yes,target variable is categorical

**Comment by philipperemy** [comment]
@rt3722 how about 
- replacing the last Dense(1) by Dense(num_classes, activation='softmax')
- using a categorical_crossentropy loss instead of mae/mse?
It will work for target variable as categorical

**Comment by rt3722** [comment]
Thank you.

**Comment by philipperemy** [comment]
@rt3722 great!
